packages/teams-js/src/internal/communication.ts

```diff
@@ -654,6 +658,21 @@ function retrieveMessageUUIDFromCallback(
   }
   return undefined;
 }
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+function removeMessageHandlers(
+  message: MessageResponse,
+  callbackId: MessageUUID,
+  map: Map<MessageUUID, unknown>,
+): void {
+  map.delete(callbackId); // need to make sure delete works since this might have by-value problems
```

Comment:
This comment suggests that you still need to look into this

Reply:
Actually, looked into this. I forgot to remove it. Thanks for catching that!